app/init.php

```diff
@@ -79,6 +79,7 @@
 const APP_STORAGE_CACHE = '/storage/cache';
 const APP_STORAGE_CERTIFICATES = '/storage/certificates';
 const APP_STORAGE_CONFIG = '/storage/config';
+const APP_STORAGE_READ_BUFFER = 20 * (1024 * 1024); //20MB other names `APP_STORAGE_MEMORY_LIMIT`, `APP_STORAGE_MEMORY_BUFFER`, `APP_STORAGE_READ_LIMIT`, `APP_STORAGE_BUFFER_LIMIT`
```

Comment:
Where is this used ? / What do we use it for ? Might give some ideas on which name applies better

Reply:
This is used when we want to read a file into memory, any file larger than 20 mb will be streamed however any file less than 20mb in size can be loaded at once into memory